client/my-sites/subscribers/components/subscriber-details/subscriber-details.tsx

```diff
@@ -88,28 +116,16 @@ const SubscriberDetails = ( {
 						{ subscriptionPlans &&
 							subscriptionPlans.map( ( subscriptionPlan, index ) => (
 								<div className="subscriber-details__content-value" key={ index }>
-									{ subscriptionPlan.title ? `${ subscriptionPlan.title } - ` : '' }
+									{ ! subscriptionPlan.is_gift && subscriptionPlan.title
```

Comment:
Should this also be included in the `SubscriberRow` component? I was expecting to see the gifts in the subscribers list (not just the profile) as you showed in your screenshot but was unable to find this in the calypso live links.

Reply:
@chrismccluskey You definitely should see it in the subscriber's list.
Are you sure:
- you are sandboxing
- you have applied D135764-code  on your sandbox?

Reply:
<img width="887" alt="Screenshot 2024-02-01 at 16 27 10" src="https://github.com/Automattic/wp-calypso/assets/790558/646fc373-6cb9-4c37-a3d2-706fbbad3eba">

Reply:
<img width="1609" alt="Screenshot 2024-02-01 at 17 13 21" src="https://github.com/Automattic/wp-calypso/assets/790558/571821bb-6c7f-41d1-a24d-e9a52fede758">

Reply:
> you are sandboxing
> you have applied [D135764-code](https://href.li/?https://code.a8c.com/D135764) on your sandbox?
> you are using define( 'USE_STORE_SANDBOX', true ); if necessary?

Yes to all three of those. You I am only not seeing it in the subscribers list.

<img width="1109" alt="Screenshot 2024-02-01 at 11 21 35 AM" src="https://github.com/Automattic/wp-calypso/assets/3662616/ea94b37f-8411-4ef7-afbb-3f1a465859a4">

I can see it when viewing the profile:

<img width="1097" alt="Screenshot 2024-02-01 at 11 21 48 AM" src="https://github.com/Automattic/wp-calypso/assets/3662616/7ade33db-98ff-4c1e-a546-a55a487f1820">

Reply:
Actually I tested it and it works fine, this one has a Gift AND a subscription:

![Uploading Screenshot 2024-02-02 at 15.19.42.png…]()

Reply:
> Yes to all three of those. I am only not seeing it in the subscribers list:

I am not sure... Does you site has the `MEMBERSHIPS_GIFT_COUPONS_STICKER` sticker?